### PR TITLE
LoKI: Create a new Tab for Experiment Configuration

### PR DESCRIPTION
**Pull request by umithardal**
This PR introduces a new tab in the LoKI GUI, called `Experiment Configuration`. This new tab currently holds `Aperture` and `Detector Offset` values. The corresponding code base has been initialised however the tab has no functionality at this time.

Upon a discussion with the instrument scientists, the requested functionalities shall be added.

### CODE REVIEW

**Comment by kmurica** [comment]
LGTM! I think the code is clean and easy to follow. @ebadkamil could you confirm it looks ok in Linux as well?

@umithardal just one very minor thing though:
Copyright (c) 2009-2020 by the NICOS contributors (see AUTHORS) 
I think we should have the years 2009-2021 here.

**Review by kmurica — APPROVED** [review]
LGTM.
Fix the year 2009-2020 to 2009-2021 in header though.
Check with Ebad it still looks fine in Linux.

**Comment by umithardal** [comment]
Thanks. I will make some changes still so made it a draft. Will fix the year.

**Comment by ebadkamil** [comment]
This is how it looks on linux. Hopefully same on mac too.

Screenshot from 2021-03-25 07-27-49

**Comment by umithardal** [comment]
@ebadkamil @kmurica I have made a small change. Could you please check if it is ok (keeping in mind that this tab will be better after I got some real feedback from the instrument scientists)? After that I will merge.

**Comment by ebadkamil** [comment]
@umithardal yes it looks fine to me.

File: nicos_ess/loki/gui/experiment_conf.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        Panel.__init__(self, parent, client, options)
+        loadUi(self, findResource('nicos_ess/loki/gui/ui_files/exp_config.ui'))
+        
+        self.experiment_frame = QFrame(self)
+
+        self.holder_info = options.get('holder_info', [])
+        self.instrument = options.get('instrument', 'loki')
+        self.initialise_connection_status_listeners()
+
+    def initialise_connection_status_listeners(self):
```
Comment (ebadkamil) [inline nicos_ess/loki/gui/experiment_conf.py:46]: The code in this method is repeated in experiment_conf.py, sampleconf.py and scripbuilder.py. May be we can add this method in the Panel class and override `on_client_connected` and `on_client_disconnected` in inherited classes.

But in a different PR, not here.

**Review by ebadkamil — APPROVED** [review]

File: nicos_ess/loki/gui/experiment_conf.py (hunk 1 above)
Comment (umithardal) [reply nicos_ess/loki/gui/experiment_conf.py:46]: Sure. Nice idea. Let me open a ticket about that.

Comment (umithardal) [reply nicos_ess/loki/gui/experiment_conf.py:46]: Just did. ECDC-2461. I will do that.

**Pull request merged**